content/docs/sidebar.json

```diff
@@ -626,10 +626,6 @@
         "label": "Get Started",
         "slug": "get-started"
       },
-      {
-        "label": "DVCLive with DVC",
-        "slug": "dvclive-with-dvc"
-      },
```

Comment:
This is a major entry point now. Do we want to burry it under How to/ Integrate with DVC checkpoints? If so let's add a redirect please 🙂 (lmk if you need help with that).

Reply:
Will add a redirect to the current get started. 

I believe the main point of the old page was not about checkpoints but about the need of creating a `dvc.yaml`, which I try to better clarify as a requirement in the current get started